app/views/onboarding/EnableExposureNotifications.tsx

```diff
@@ -58,17 +72,23 @@ export const EnableExposureNotifications = (): JSX.Element => {
           </View>
 
           <View style={styles.footerContainer}>
-            <Button
```

Comment:
Does removing this allow you to remove the Theme above?

Reply:
probably! will leave for another commit though.